datatypes.h

```diff
@@ -987,14 +987,14 @@ typedef enum {
 } CAN_PACKET_ID;
 
 // VPT
-#define VPT_CAN_SYS_ID 0xAA00
+#define VPT_CAN_SYS_ID 0xAB
```

Comment:
Just confirming that you wanted to change to `0xAB` from `0xAA`.

Reply:
Yes, I wanted to change the ID since the data structure used to communicate changed. This is essentially like a semver, which prevents communication between incompatible software versions of the ACUs and FCU.